bip-0043.mediawiki

```diff
@@ -53,6 +55,17 @@ and calling themselves BIPxx compatible, we suggest that software which needs
 only a limited structure should describe such structure in another BIP and use
 different "purpose" value.
 
+===Purpose 43===
+
+Purpose code 43 is reserved for usage by non-Bitcoin cryptocoins.
+In this case, the level immediately below the purpose code is hardened derivation using the index for the cryptocoin specified by [[https://github.com/satoshilabs/slips/blob/master/slip-0044.md|SLIP-0044]].
+The details of levels below this coin type are to be entirely decided by the standard process of the specific cryptocoin, but it is recommended that the very next level be a sub-purpose code with a similar use as this BIP's purpose code.
+
+So that the recommended use for non-Bitcoin cryptocoins is:
+
+<pre>
+m / 43' / coin_type' / subpurpose' / ...
+</pre>
```

Comment:
Could this PR get merged if someone pulls out this specification to some other place, such that you do no more than reserving 43 here?